File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet<Integer> dup = new HashSet<>();
        for(int i = 0; i < nums.length; i++){
            dup.add(nums[i]);
        }
        if(nums.length == dup.size()){
            return false;
        }
        return true;
    }
}

File: Data Structures & Algorithms/single-number/submission-0.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        list = {}
        for i in nums:
            if i in list:
                list[i] += 1
            else:
                list[i] = 1
        found_key = None
        for key, value in list.items():
            if value == 1:
                found_key = key
        return found_key